### PR TITLE
Added request reviewers function within GHPullRequest.

### PR DESCRIPTION
**Pull request by twcurrie**
Low hanging fruit:  https://developer.github.com/v3/pulls/review_requests/#create-a-review-request

### CODE REVIEW

File: src/main/java/org/kohsuke/github/GHPullRequest.java [hunk 1]
```diff
@@ -345,6 +346,12 @@ public GHPullRequestReviewComment createReviewComment(String body, String sha, S
                 .to(getApiRoute() + COMMENTS_ACTION, GHPullRequestReviewComment.class).wrapUp(this);
     }
 
+    public void requestReviewers(List<GHUser> reviewers) {
+        return new Requester(root).method("POST")
+                .with("reviewers", toLogins(reviewers))
```
Comment (markkolich) [inline src/main/java/org/kohsuke/github/GHPullRequest.java]: @twcurrie any plans to also support `team_reviewers`?

https://developer.github.com/v3/pulls/review_requests/#create-a-review-request

**Comment by twcurrie** [comment]
Sure, is this a blocker?  (Are you even able to merge this?)

File: src/main/java/org/kohsuke/github/GHPullRequest.java (hunk 1 above)
Comment (twcurrie) [reply src/main/java/org/kohsuke/github/GHPullRequest.java]: Sure, is this a blocker? (Are you even able to merge this?)

Comment (markkolich) [reply src/main/java/org/kohsuke/github/GHPullRequest.java]: > (Are you even able to merge this?)

No, I'm simply advocating for `team_reviewers` &mdash; I was going to open my own PR for https://developer.github.com/v3/pulls/review_requests/#create-a-review-request but then I saw this one.

I think we need @kohsuke to review and merge.

**Comment by markkolich** [comment]
@kohsuke any chance we can get this merged and released?

**Comment by kohsuke** [comment]
Oy, you didn't even bother compiling this before submitting a PR...

**Comment by jgangemi** [comment]
how about adding travis or circle to build requests?

**Comment by twcurrie** [comment]
I built it locally.  Your doc's are good but pretty lacking on contributing guidelines.

**Comment by kohsuke** [comment]
Yeah I'll add CodeShip.

@twcurrie it's no big deal, but the way I noticed it is that the code didn't even compile; no semi-colon at the end, a return statement with a value for a function whose return type is `void`, ...

**Comment by twcurrie** [comment]
Oh my, I'm clearly writing too much kotlin nowadays.  It did compile in kotlin, but yes, obviously not in java...

Thanks for putting together this library - it's been a big help.

**Comment by jgangemi** [comment]
and this is why i dislike languages that try and be crafty by letting you omit things like a `;`, but i digress and don't want to thread jack 😄

**Pull request merged**